dashboard/v1.1/src/layouts/QuickInputFab.tsx

```diff
@@ -22,8 +23,12 @@ const useStyles = makeStyles(theme => ({
 }));
 
 // Floating Action Button for Quick Input
-const QuickInputFab = () => {
+const QuickInputFab = props => {
   const classes = useStyles();
+  const { location } = props;
```

Comment:
Can we use `useLocation` instead?

https://reactrouter.com/web/api/Hooks/uselocation

Reply:
@chetanyakan any particular problem with this approach is so we can go with your approach

Reply:
Let's keep this as it is, for now, we can cautiously work on stuff in `v2.0`